clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1715,6 +1723,8 @@ scheduler_tmpl_t<RecordType, ReaderType>::next_record(output_ordinal_t output,
                 record = **input->reader;
             }
         }
+        VPRINT(this, 5, "next_record_mid[%d]: from %d: ", output, input->index);
```

Comment:
Not clear what next_record_mid means. Was it for your local debugging?

Reply:
Yes, was going to just delete it, but thought it may be useful next time.  I replaced _mid with a separate description "candidate record from %d".  This is the possible next record which is examined to see if we should switch, when it is queued.  Useful to see what it is for some debugging.